content/2016.06.kinto-admin-1.0.md

```diff
@@ -0,0 +1,39 @@
+Title: Kinto-admin reaches 1.0
+Date: 2016-06-10
+Lang: en
+Url: kinto-admin-1.0
+Slug: kinto-admin-1.0
+Summary: Introducing kinto-admin, a Web UI for managing your Kinto data
+
+When choosing a database, it's a good idea to study a few important criteria: how nice is the API? Is it well documented? Will it scale? What is the tooling available? Tooling is especially important as it's what will make your life as a developer easier. Basically [kinto-admin] wants to be to Kinto what [phpMyAdmin] is to MySQL. We've mentioned kinto-admin a few times on this blog already, and today we're announcing the release of [version 1.0].
+
+![]({filename}/images/kinto-admin-logged-in.png)
+
+Kinto-admin 1.0 allows users to discover and browse the buckets, collections and records they have access to on a given Kinto server. They can also create, update and delete them, provided they have the required permissions.
+
+Collection settings are configurable from the admin, including defining their [JSON schema] and display options such as fields to render in the records list and the widgets to use to render form fields. These settings are stored in the collection metadata on the server; that's right, no need for configuration files anymore!
+
+![]({filename}/images/kinto-admin-collection-edit.png)
+
+![]({filename}/images/kinto-admin-delete-collection.png)
+
+For authentication, kinto-admin currently supports [Basic Auth], and [Firefox Account] if [kinto-fxa] is installed on the server, though we're planning to add more providers in the future.
+
+We've also added native support for [kinto-attachments], which means you can attach files to records and post them to the server when the capability is available.
+
+![]({filename}/images/kinto-admin-edit-attachment.png)
+
+We really hope you'll enjoy this new version as much as we loved building it. You can use [our hosted instance] on Github.
```

Comment:
Maybe precise here that it's only static js / css / html files, and that it's really easy to deploy anywhere else.